addons/lunch/lunch.py

```diff
@@ -318,6 +177,7 @@ def fields_view_get(self, cr, uid, view_id=None, view_type=False, context=None,
         'date': fields.date.context_today,
         'state': 'new',
         'alerts': _default_alerts_get,
+        'previous_order_details': _default_get_lunch_order_detail,
```

Comment:
why same private method *_default_get_lunch_order_detail* is call from defaults and also from  get_lunch_order_detail?